### DISCUSSION TITLE
Broadcasting event to livewire method is not listening using Laravel Echo

### DISCUSSION THREAD

**Discussion by balasoundariy** [livewire/livewire#4918] [1. Help]
In Livewire documentation https://laravel-livewire.com/docs/2.x/laravel-echo we can use below codes to listen from pusher, while broadcasting the event.

When comes to JS method, It listen the event from pusher as I mentioned below,

        Echo.private('orders').listen('OrderShipped', (e) => {
            console.log(e.order.name);
        });

When comes to Livewire method, I'm not able to listen the event from pusher using below code in livewire component. 

    public function getListeners()
    {
        return [
            "echo-private:orders.{$this->orderId},OrderShipped" => 'notifyNewOrder',
            // Or:
            "echo-presence:orders.{$this->orderId},OrderShipped" => 'notifyNewOrder',
        ];
    }
 
    public function notifyNewOrder()
    {
        $this->showNewOrderNotification = true;
    }

Can anyone help me how can I make it work in livewire method?

**Comment by francoism90** [comment]
@balasoundariy Same issue. :/
